Unity/settings

rachel92124 opened the pull request:
I added a settings menu and settings buttons to the pause menus in all of the levels
Check that the pose detection check box is unchecked when movenet is not running and is checked when it is. Let me know if you run into situations where this is not the case.
Check that the state of the settings menu and the settings on the pause menu are consistent with each other.

sydneylaiw commented:
Only comment is I think changing "In game" to "In-game" might make it more readable. Other than that, everything is functional!

rachel92124 commented:
I changed to in-game, made hints off by default
I also added an extra state beyond T and F to the MovenetConnected playerpref so I could know when Movenet is starting but not running yet. This is so Movenet wont be closed when settings is opened but Movenet is not fully running yet.

sydneylaiw commented:
re-reviewed this just now. everything looks good and ready to merge to me, waiting for @anreeta to confirm
